Celeste.Mod.mm/Mod/Meta/MapMeta.cs

```diff
@@ -382,6 +405,38 @@ public ModeProperties Convert()
             }
             area.Mode[(int) mode] = props;
         }
+
+        public void AddTo(MapMetaModeProperties other) {
```

Comment:
And and AddTo are also implemented for MapMetaModeProperties and MapMetaAudioState. It could theoretically also be implemented for the further-down meta classes, such as checkpoint meta, but those are never serialized into bins by any known level editor so it's not a priority.